notebooks/00_HubQuickStart.ipynb

```diff
@@ -1,10 +1,67 @@
 {
```

Comment:
<p>Somewhat unrelated: I think it would be helpful to give the user the option of running this locally as well (and maybe make it part of the documentation website too).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass-demo/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I see the aim of this notebook as being to run things on the hub with the `.sql` backup. I think expanding the aims will make the original goal less stable ... or it would require a lot of preface about 'download this, place it here, edit that path' etc